README.md

```diff
@@ -1,33 +1,45 @@
-Jovian NixOS
-============
+NixOS on the Steam Deck
+=======================
 
-> **Jovian**  
-> “Relating to [...] Jupiter or the class of [...] which Jupiter belongs.”
+A set of packages and configurations for running NixOS on the [Steam Deck](https://www.steamdeck.com).
+This repo is also useful for obtaining a Deck-like experience on other X86-64 devices.
```

Comment:
```suggestion
This repo is also useful for obtaining a Deck-like experience on other x86_64 devices.
```

Format used by e.g.  `uname` and by convention system types in Nix and Nixpkgs.

Or even `x86_64` (with backticks to make it `<code />`)